lib/menu.rb

```diff
@@ -0,0 +1,20 @@
+class Menu
+  attr_reader :list
+
+  def initialize
+    @list = VEGETARIAN_MENU
+  end
+
+  def print_menu
+    list.each do | dish, price |
+      dish_list = (dish.to_s).ljust(25)
+      price_list = "£#{price.to_s}".rjust(25)
+      puts dish_list + price_list
```

Comment:
It's more idiomatic to use string interpolation `"#{}"` than concatenation `a + b`.